lib/reporters/junit/index.js

```diff
@@ -31,8 +31,8 @@ JunitReporter = function (newman, reporterOptions) {
         root.att('tests', _.get(newman, 'summary.run.stats.tests.total', 'unknown'));
 
         cache = _.transform(report, function (accumulator, execution) {
-            accumulator[execution.id] = accumulator[execution.id] || [];
-            accumulator[execution.id].push(execution);
+            accumulator[execution.item.id] = accumulator[execution.id] || [];
```

Comment:
@kunagpal Isn't this assignment wrong now?
`accumulator[execution.id]` will be always null.

Reply:
`execution.id` is actually `item.id` 🤔
https://github.com/postmanlabs/newman/blob/develop/lib/run/summary.js#L152